README.md

```diff
@@ -465,6 +399,126 @@ extensions such as `import com.klaviyo.pushFcm.KlaviyoRemoteMessage.body` to acc
 We also provide an `Intent.appendKlaviyoExtras(RemoteMessage)` extension method, which attaches the data to your
 notification intent that the Klaviyo SDK requires in order to track opens when you call `Klaviyo.handlePush(intent)`.
 
+## In App Forms
+
+[In-app forms](https://help.klaviyo.com/hc/en-us/articles/34567685177883) are messages displayed to mobile app 
+users while they are actively using your app. You can create new in-app forms in a drag-and-drop editor in the 
+Sign-Up Forms tab in Klaviyo. Follow the instructions in this section to integrate forms with your app. The SDK will
+display forms according to targeting and behavior settings and collect delivery and engagement analytics automatically.
```

Comment:
Slight tweak, but I feel like this reads a little better - "according to **their** targeting and behavior settings"

Reply:
I think iOS has the same verbiage, i can get both patched.

Reply:
```suggestion
display forms according to their targeting and behavior settings and collect delivery and engagement analytics automatically.
```